react/SearchAction.tsx

```diff
@@ -19,21 +19,26 @@ declare var window: {
 }
 
 const SearchAction = () => {
+  const runtime = useRuntime()
   const protocol = 'https'
   const hostname = canUseDOM ? window.location.hostname : global.__hostname__
-  const rootPath = canUseDOM ? window.__RUNTIME__.rootPath : global.__RUNTIME__.rootPath
+  const rootPath = canUseDOM
+    ? window.__RUNTIME__.rootPath
+    : global.__RUNTIME__.rootPath
 
   const baseUrl = `${protocol}://${hostname}${rootPath || ''}/`
```

Comment:
I would remove this trailing '/' and add it to the searchTerm so it's easier to understand it's value in the store's setting input